.changeset/selfish-gorillas-confess.md

```diff
@@ -0,0 +1,6 @@
+---
+"@sap-ux/ui-components": major
+"@sap-ux/types": minor
```

Comment:
`@sap-ux/types` is not changed here? `types/package.json`

For `@sap-ux/ui-components` Wouldn't patch / minor be more appropriate?
Is it technically a breaking change?
`UIBooleanSelect` is only used in data editor as far as I am aware.

Reply:
`@sap-ux/types` was changed cause both `UIColumn` and `ColumnControlType` was changed, added a optional field.

Yes, it is only used in the data editor, but shouldn't that then still be marked as a breaking change, cause previous api has been removed and this is an open source project?

Reply:
@Klaus-Keller @devinea removed the change marker from `@sap-ux/types` and made the other changes a minor bump by adding support for both control types